README.md

```diff
@@ -1,22 +1,78 @@
-# GetIt
+<div align="center">
 
-## Build Setup
+# 🛒 GETIT
+</div>
+Streamlined college e-commerce: Simplifying merch buying and selling for students.
+
+
+## ✨FEATURES
+
+- Clubs can sell
+- Buy the merch you like
+- Automatic payments for sellers no need for manual verification
+- Seller reports
+
+
+
+
+---
+## 📱 Tech Stack 
+
+[![My Skills](https://skills.thijs.gg/icons?i=vue,nuxt,nodejs)](https://skills.thijs.gg)
+### FRONTEND:
+- [VUE](https://vuejs.org/guide/introduction.html)
+- [NUXT](https://nuxt.com/docs)
+
+### BACKEND: 
+
+TO BE UPDATED
+
+
+## 🔰 GETTING STARTED
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### [![My Skills](https://skills.thijs.gg/icons?i=nodejs)](https://skills.thijs.gg)PREREQUISITS
+
+The following software is required to be installed on your system:
+
+* Node 16.x
+* Npm 8.x
+
+Type the following commands in the terminal to verify your node and npm versions
+
+```bash
+node -v
+npm -v
+```
+
+### ![NPM](https://img.shields.io/badge/NPM-%23CB3837.svg?style=for-the-badge&logo=npm&logoColor=white) BASIC SETUP
+
+
+#### Follow the steps mentioned below to setup the project locally on your computer
+
+1. Fork the repository by clicking on `Fork` option on top right of the main repository.
+2. Open Command Prompt/Terminal on your local computer.
+3. Clone the forked repository by adding your own GitHub username in place of `<username>`.
 
 ```bash
-# install dependencies
-$ npm install
+    git clone https://github.com/<username>/Getit/
+```
+4. Navigate to the Getit directory.
 
-# serve with hot reload at localhost:3000
-$ npm run dev
+```bash
+    cd Getit
+```
 
-# build for production and launch server
+5. Build Setup
+```
+# build for production and launch server:3000
 $ npm run build
 $ npm run start
 
 # generate static project
 $ npm run generate
```

Comment:
Hey @RISHIKESHk07 Can you please also include instructions to build the local development env for dev setup ?

- npm install
- npm run dev

Reply:
Hey @RISHIKESHk07 , Either add this section in Build for production guidelines, or remove this, since it may confuse the first time users who are trying to setup the env locally.